frontend/pages/_app.tsx

```diff
@@ -1,16 +1,28 @@
 import '../styles/globals.css';
 import type { AppProps } from 'next/app';
 import { ThemeProvider } from '@mui/material/styles';
+//import image from 'public/bg7.png';
 import Header from '@/components/utils/Header';
 import CustomTheme from '@/components/utils/CustomTheme';
 import { AppProvider } from '@/components/Context/TopLevelContext';
 
 function MyApp({ Component, pageProps }: AppProps) {
+  const myStyle={
+    backgroundImage: 
+"url('https://raw.githubusercontent.com/uwasystemhealth/IndEAA/develop/client/assets/img/bg7.jpg')",
```

Comment:
Make this image part of the public asset and get it from there instead from a url